### ISSUE TITLE
Unified Search Dialog: Resurrect Ctrl+T Keyboard Shortcut

### ISSUE THREAD

**Issue by metal450** [giuspen/cherrytree#1829]
Thanks very much for https://github.com/giuspen/cherrytree/issues/513 :)

Just one small ask: although the unified search dialog is super appreciated, I do think it's still useful to retain the Ctrl+T keyboard shortcut to search in names only, which seems gone in the current build. That way, if you happen to know you're looking for a node name (which is much, much faster), you can still do it without needing to use the mouse to alter the checkboxes by hand.

So basically,
Ctrl+Shift+F: Brings up the dialog with both content+node names options selected (broadest search)
Ctrl+T: Brings up the (same unified) dialog with only node names selected by default

Does that make sense? So the new feature would add the ability to simultaneously search in both names+content, but doesn't lose the ability to search just in names, if desired for its speed.
